Dockerfile

```diff
@@ -16,7 +38,7 @@ RUN apt-get update && apt-get install -y \
     python3-dev \
     python3-pip \
     python3-venv \
-    python-software-properties \
+    r-base \
     software-properties-common \
     wget \
     zip && \
```

Comment:
it looks like this package list has been copied from somewhere and we don't need the majority of these packages. E.g. `bc` is a calculator. I suggest shrinking it down to:
```
build-essential \
curl \
groff \
python3 \
python3-pip \
r-base \
software-properties-common
```

Note, `groff` is required by awscli

Reply:
updated docker file as suggested.